tests/test_routes.py

```diff
@@ -126,13 +126,13 @@ def test_upload_xlsx_uncaught_validation_error(requests_mock, example_pre_ingest
 
     assert response.status_code == 200
     assert (
-        f'Contact us at <a href="mailto:{service_email}">{service_email}</a>. Do not '
-        "send your return or any attachments using the help email."
+        f'Your error code is [XXXX]. Please email us on <a href="mailto:{service_email}">{service_email}</a> and '
```

Comment:
Given we don't have a way to generate this error code, I think we may need to  revert and decope this?

Reply:
I agree that we don't have the designs or a flow for generating these codes, at this stage seems unlikely it will get done for October

Reply:
Happy to revert or let someone else pick this up prehaps when completing SMD-142 that is supposed to build on this feature